VagrantFile

```diff
@@ -0,0 +1,67 @@
+#to UNIX EOL
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# specifies the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+
+# This file is built from the great work of Cristina Muñoz over at
+# https://github.com/PermanentOrg/devenv
+
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search or even make your own using
+  # the steps outlined at https://www.vagrantup.com/vagrant-cloud/boxes/create.
+  config.vm.box = "generic/debian10"
+  config.vm.box_version = "3.1.10"
+  config.vm.define "lever-for-change"
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  config.vm.box_check_update = true 
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine. In the example below,
+  # accessing "localhost:8080" will access port 80 on the guest machine.
+
+  # Create a private network, which allows host-only access to the machine
+  # using a specific IP.
+  config.vm.network "private_network", ip: "192.168.33.11"
```

Comment:
Meaning the host access the virtual machine with the VM having 192.168.33.11 as its IP address?  What happens if some other device on the host's network already has that as its dynamically assigned address?  Or the host itself does?

(Assuming there's a reasonable answer to this question, that answer should go in the comment in the file.)

Reply:
Correct -- this is relic from the original project (though I did increment the IP by 1 to avoid collision for devs working on both projects...).

The best-and-least-engineered-solution here would be to just let vagrant use DHCP and output the IP at the end of the script.  In fact... I'm going to go ahead and try that.

More information here: https://www.vagrantup.com/docs/networking/private_network

Reply:
Update on this: one could be *extra fancy*, specify a hostname and then automatically update the host file on the dev machine  but that feels both overly invasive and also relies on an additional vagrant plugin so not going to do that here.

I've added a little shell script for ip lookup (vagrant itself wasn't making it easy / something is strange with inline provision scripts).  Will push that to the branch tomorrow.

Reply:
Cool, thanks!